spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncherApplication.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.launcher;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+/**
+ * Spring Boot launcher application
+ *
+ * @author Marius Bogoevici
+ */
+@SpringBootApplication
+@ComponentScan(basePackageClasses = ModuleLauncherApplication.class)
+public class ModuleLauncherApplication {
+
+	private static final Log log = LogFactory.getLog(ModuleLauncherApplication.class);
+
+	public static void main(String[] args) throws Exception {
+		try {
+			SpringApplication.run(ModuleLauncherApplication.class, args);
+		} catch (Exception e) {
+			log.error("Couldn't launch module: ", e);
```

Comment:
no module string follows the colon (also maybe "Failed to launch module" would be better).